Test Plan Operations Examples/README.md

```diff
@@ -0,0 +1,154 @@
+# Test Plan Operations Example
+
+This example demonstrates how to define a test plan template with execution
+actions that can be used to execute a test plan on a target system.
+
+## Test plan templates
+
+Test plan templates appear in the Work Order UI when creating a new test plan
```

Comment:
I would change to "A test plan template can be specified to provide product specific default values when creating a new test plan."